spec/controllers/v2/application_controller_spec.rb

```diff
@@ -4,7 +4,7 @@
 
 describe V2::ApplicationController do
   describe 'rbac_allowed?' do
-    let(:user) { User.new }
+    let(:user) { FactoryBot.build(:user) }
```

Comment:
@xmicha82 you were not using the factory here, this was causing a warning in RSpec